.codegen/workspaces.go.tmpl

```diff
@@ -17,7 +17,9 @@ type WorkspaceClient struct {
 	{{end}}{{end}}
 }
 
-// NewWorkspaceClient creates new Databricks SDK client for Workspaces or 
+var ErrNotWorkspaceClient = errors.New("invalid Databricks Workspace configuration")
```

Comment:
It doesn't, but given that 1) it's there for the account client as well, and 2) it would require adding another file, I figured I might as well stick to what's there.